mergify_cli/ci/cli.py

```diff
@@ -76,3 +82,84 @@ async def junit_upload(  # noqa: PLR0913, PLR0917
             click.style(f"Error uploading JUnit XML reports: {e}", fg="red"),
             err=True,
         )
+
+    if spans:
+        failing_spans = [
+            span
+            for span in spans
+            if span.status.status_code == opentelemetry.trace.StatusCode.ERROR
+            and span.attributes is not None
+            and span.attributes.get("test.scope") == "case"
+        ]
+        if not failing_spans:
+            return
+
+        await check_failing_spans_with_quarantine(
+            api_url,
+            token,
+            repository,
+            [fspan.name for fspan in failing_spans],
+        )
+
+
+@tenacity.retry(
+    wait=tenacity.wait_exponential(multiplier=0.2),
+    stop=tenacity.stop_after_attempt(5),
```

Comment:
I would be more conservative by reducing the number of attempts, but increasing the backoff

Reply:
I don't think we need to be conservative that much, especially since at some point this will be in the CI, so waiting too much can be annoying and costly imo.

We'll still be able to fine-tune this retry behavior later, so i'll let it as-is for now if it's not blocking for you.